### ISSUE TITLE
Error status not always an integer, sometimes a string

### ISSUE THREAD

**Issue by jstayton** [elastic/elasticsearch-js#325]
Most (all?) of the errors produced by the library itself have an integer `status`. For example,

```js
client.getSource({
  index: 'my_index',
  type: 'my_type',
  id: 'not_found'
})
.catch(function(error) {
  console.log(error.status);
});
```

outputs the integer `404`.

When creating an error directly, however, it's a string. For example,

```js
var errors = require('elasticsearch').errors;
var notFound = new errors.NotFound();

console.log(notFound.status);
```

outputs the string `'404'`.

Not a big deal, but it is somewhat annoying when you're doing something like `error.status === 404`.

**Comment by spalger** [comment]
Good point

**Comment by spalger** [comment]
Closed by 0e3fe1586fd867275fd0e9abb7126724be3d3761

Will release soon

**Comment by jstayton** [comment]
Thanks! :+1:
